js-modules/regexpPrecederPatterns.pl

```diff
@@ -58,11 +66,3 @@
 $pattern .= ")\\\\s*'";  # matches at end, and matches empty string
 
 print "$pattern;\n";
-
-print "
-// CAVEAT: this does not properly handle the case where a regular
```

Comment:
yes, that way it shows up above the regex it's talking about when included in the final bundle.